src/main/webapp/WEB-INF/index.html

```diff
@@ -0,0 +1,36 @@
+<!doctype html>
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org">
+<head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/js/bootstrap.bundle.min.js" integrity="sha384-b5kHyXgcpbZJO/tY9Ul7kGkf1S0CWuKcCD38l8YkeH8z8QjE0GmW1gYU5S9FOnJ0" crossorigin="anonymous"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+<!--    <script src="collector.js" type="text/javascript"></script>-->
+<!--    <script th:src="@{/collector.js}" type="text/javascript"></script>-->
+
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
+    <!--    <link href="style.css" rel="stylesheet" type="text/css">-->
+    <!--    <link th:href="@{/style.css}" rel="stylesheet" type="text/css">-->
+
+    <title>SONET SLOVAKIA</title>
+</head>
+    <body>
+        <h1>Address</h1>
+
+        <header id="header"></header>
+        <div id="content">
+            <p>Country: <span th:text="${country}"></span></p>
```

Comment:
tie nazvy premennych by som zmenil podla konvenccie vyssie spomenutej src/main/java/Database/Tables/T_Address.java kde je T_Adress.Country; a pokial sa to tu neda vyuzit tak aspon podla toho.

Reply:
Z templatu sa nedaju uchopit java konstanty, na to je tam ten unikatny key, jedina referencia ktoru vies v htmlku pouzit.